### PR TITLE
docs: update nuxt documentation and branding

### PR DESCRIPTION
**Pull request by danielroe**
We love Tailwind CSS ❤️ This PR contains a few changes to the Nuxt documentation. It:

1. updates the name of `Nuxt.js` to `Nuxt`
2. updates the Nuxt logo
3. moves Nuxt 3 (the current stable release) to be the first set of instructions visible on the Nuxt docs
4. updates the instructions to use the [`@nuxtjs/tailwindcss` module](https://tailwindcss.nuxtjs.org/) (maintained by @atinux) and our new CLI

I hope these are all welcome. Feel free to ping me if there's anything I can do or you have any questions 🙏

### CODE REVIEW

**Comment by thecrypticace** [comment]
Hey, thanks for the PR. We generally like to keep all the guides roughly the same — including not using framework-specific packages unless absolutely necessary. However, we do see that there is some value in providing those and are going to update the guide to list both installation methods (as well as remove Nuxt 2).

I pushed some changes but am going to iterate on it more tomorrow.

File: src/pages/docs/guides/nuxtjs.js [hunk 1]
```diff
@@ -84,6 +82,7 @@ let tabs = [
 >     "./pages/**/*.vue",
 >     "./plugins/**/*.{js,ts}",
 >     "./nuxt.config.{js,ts}",
```
Comment (danielroe) [inline src/pages/docs/guides/nuxtjs.js:84]: We should remove this as it causes a very long slow-down on app start.

```suggestion
```

**Comment by thecrypticace** [comment]
Sorry "tomorrow" turned into "some time next week" 😂

Merged — should be deployed in the next 5 minutes or so.

**Comment by danielroe** [comment]
Thank you ❤️

**Comment by Atinux** [comment]
Thanks!

**Pull request merged**